### PR TITLE
Ru: arena

### PR DESCRIPTION
**Pull request by Sinitsaa**

### CODE REVIEW

**Comment by alama** [comment]
Mind if I fix your Conflict?

**Comment by Sinitsaa** [comment]
Ok

File: translated files/Story/Episode 1/Unsorted/ra_010110.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+ra_010110_0010#0,"""Эй. Ты не видел тут редкого<br>оружия?"""
+ra_010110_0020#0,"""Ох, нет? Это для моего<br>друга-оружейника, потерявшего<br>себя."""
+ra_010110_0030#0,"""Я хочу попросить его сделать<br>мне оружие, но, похоже, он<br>просто сдался."""
+ra_010110_0040#0,"""Думал, если покажу ему особое<br>оружие, к нему вернется былой<br>дух."""
+ra_010110_0050#0,"""Оно не выглядит как оружие,<br>просто что-то с потенциалом.""""
```
Comment (alama) [inline translated files/Story/Episode 1/Unsorted/ra_010110.csv]: Extra `"` at the end of line

File: translated files/Story/Episode 1/ra_010250.csv [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-ra_010250_0070#0,"""チッ、気勢が削がれたか。間の悪いヤツめ。"""
-ra_010250_0075#0,"""……まァ、いい。<br>機会はいつでもあるからな。"""
-ra_010250_0080#0,"""じゃあなァ、ゼノ。<br>甘ちゃんは甘ちゃんらしく、群れてピーピー慰め合ってな！"""
-ra_010250_0090#0,"""……それでは、失礼します。"""
+ra_010250_0010#0,"""Не связывайся со мной!"""
+ra_010250_0020#0,"""Хорошо, Зено! Дракой с тобой...<br>я смогу порадовать себя!"""
+ra_010250_0030#0,"""Зено, успокойся!"""
+ra_010250_0040#0,"""Ей, Мельфонсина! Успокой эту<br>маньячину!"""
+ra_010250_0050#0,"""Я не буду, я знаю, кого вы<br>имеете в виду."""
+ra_010250_0060#0,"""<%me>..."""
```
Comment (alama) [inline translated files/Story/Episode 1/ra_010250.csv]: This string should be `"""<%me>……"""`

**Comment by alama** [comment]
Any more changes?

**Comment by Sinitsaa** [comment]
Next week.

**Pull request merged**